Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -307,7 +328,7 @@ def save_caskfile
     end
 
     def uninstall
-      odebug "Hbc::Installer#uninstall"
+      oh1 "Uninstalling Cask #{@cask}"
```

Comment:
I don't think we ever use `oh1` for Homebrew-Cask, so let's use `ohai` here, too.